packet.go

```diff
@@ -690,8 +698,12 @@ func (p *sshFxpOpenPacket) UnmarshalBinary(b []byte) error {
 		return err
 	} else if p.Pflags, b, err = unmarshalUint32Safe(b); err != nil {
 		return err
-	} else if p.Flags, _, err = unmarshalUint32Safe(b); err != nil {
+	} else if p.Flags, b, err = unmarshalUint32Safe(b); err != nil {
 		return err
+	} else if p.Flags&sshFileXferAttrPermissions != 0 {
```

Comment:
Where is the `server.go` and `request_server.go` code to handle this expanded feature set? This is one of the further complications to this sort of a PR: this is not simply a client-side concern, there is also a concern about how to handle the server side of the feature.

Reply:
Understood, but note that this is a broader issue, as other client implementations can also send open packets with attributes that the servers will ignore.

Reply:
True.